linter/src/main/java/com/facebook/coresql/linter/lint/LintingVisitor.java

```diff
@@ -12,12 +12,12 @@
  * limitations under the License.
  */
 
-package com.facebook.coresql.lint;
+package com.facebook.coresql.linter.lint;
 
+import com.facebook.coresql.linter.warning.WarningCode;
```

Comment:
I see in your other PR that you are adding warnings from rewriter. In that case should warning belong to linter?

Reply:
I intend this PR to contain the correct structure of all linter/warning code. After merging this branch, I'm going to merge it into the other branches (which may be confusing since -- like you mention -- there is a conflict at the moment).

Done